game/forms.py

```
@@ -142,4 +142,39 @@ def clean_content(self):
+
+        if avatar.size > self.MAX_SIZE_BYTES:
+            raise ValidationError(
+                "Avatar file is too large. Maximum allowed size is 5 MB."
+            )
+
+        content_type = getattr(avatar, "content_type", "")
+        if content_type not in self.ALLOWED_MIME_TYPES:
+            raise ValidationError(
+                "Unsupported image format. "
+                "Please upload a PNG, JPG/JPEG, or WEBP file."
+            )
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)